src/main/scripts/ncm-ncd

```diff
@@ -119,13 +119,19 @@ sub app_options()
         },
 
         {
+            NAME    => 'nodepsnoerrors',
+            HELP    => 'errors from broken (pre/post) dependencies in configure are downgraded to warnings (implies --nodeps)',
+            DEFAULT => undef
+        },
+
+        {
             NAME    => 'skip:s',
             HELP    => 'skip one component (only to be used with --all)',
             DEFAULT => undef
         },
 
         {
-            NAME    => 'autodeps',
+            NAME    => 'autodeps!',
```

Comment:
in pure `Getopt::Long`, it means the a negation of form `--no-<optioname>` or `--no<optionname>` can disable it (but it seems to already work somehow). i wasn't aware of it and i want to make it explicit